src/lib/layer.js

```diff
@@ -284,6 +290,16 @@ export default class Layer {
   }
 
   /**
+   * Returns the picking color of the object at 'selectedObjectIndex' prop.
+   * @return {Array} - the picking color or null if 'selectedObjectIndex is invalid.
+   */
+  selectedObjectPickingColor() {
+    if (this.props.selectedObjectIndex < 0) {
+      return null;
```

Comment:
OK I can accept that.

The counter argument would be that it feels like we have an if statement returning null here, and another if statement checking for null in the shader module. And `encodePickingColor` already handles `index = -1`. So a bunch of additional logic that ultimately cancels out. But maybe this is still easer for the reader to follow.